src/main/steps/applicant1/finalising-your-application/content.ts

```diff
@@ -6,13 +6,15 @@ import { isFieldFilledIn } from '../../../app/form/validation';
 import { CommonContent } from '../../common/common.content';
 import { generateContent as columnGenerateContent } from '../hub-page/right-column/content';
 
-const en = ({ isDivorce, partner, userCase }: CommonContent) => ({
+const en = ({ isDivorce, partner, userCase, isJointApplication }: CommonContent) => ({
   title: `Do you want to ${isDivorce ? 'finalise your divorce' : 'end your civil partnership'}?`,
   line1: `Your ${
     isDivorce ? 'marriage' : 'civil partnership'
   } will be legally ended after the final order is made. This might affect your finances.`,
   warningText: `If you have not finished negotiations about your money, property or other assets then you should seek legal advice before finalising
-  ${isDivorce ? 'your divorce' : 'ending your civil partnership'}`,
+  ${isDivorce ? 'your divorce' : 'ending your civil partnership'} ${
+    isJointApplication ? 'If you want to settle your finances first, then save and sign out.' : ''
```

Comment:
this seems like the start of a new sentence but theres no period before and its not a new line.

Reply:
Hi @velimd added a period after the first line.